assets/charts/control-plane/kube-apiserver/templates/kube-apiserver-ds.yaml

```diff
@@ -24,6 +24,8 @@ spec:
       annotations:
         checkpointer.alpha.coreos.com/checkpoint: "true"
         seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+        # Automatically rolls update when secret changes.
```

Comment:
This should go into other commit and have e2e tests.

Reply:
Will rework commits.

Reply:
I'll mark that as resolved.